ext/civicrm_admin_ui/managed/SavedSearch_Contact_Search.mgd.php

```diff
@@ -3,6 +3,32 @@
 use CRM_CivicrmAdminUi_ExtensionUtil as E;
 
 return [
+  // using a temporary alternate path to avoid breaking legacy path
+  [
+    'name' => 'Navigation_Find_Contacts',
+    'entity' => 'Navigation',
+    'cleanup' => 'unused',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'domain_id' => 'current_domain',
+        'label' => E::ts('Find Contacts'),
+        'name' => 'Find Contacts',
```

Comment:
Maybe doesn't matter, but should this be `Find_Contacts`?

Reply:
"Find Contacts" is the existing name. No choice if we want to update existing path.